test/letters/01-authed.e2e.spec.js

```diff
@@ -83,6 +83,43 @@ module.exports = E2eHelpers.createE2eTest(
       .click('div.step-content > p:nth-child(3) > a') // link to go back to confirm-address
       .expect.element('.city-state').to.be.present.before(Timeouts.normal);
 
+    // -- Go to letters list -- //
+
+    client
+      .click('.view-letters-button')
+      .assert.urlContains('/letters/letter-list')
```

Comment:
Is the new URL available instantly or does there need to be a wait somewhere in there? Not sure how that works with Router.

Reply:
It should be available instantly. If this fails suspiciously later, we can reassess. :man_shrugging: